packages/eas-cli/src/credentials/ios/appstore/ensureAppExists.ts

```diff
@@ -16,20 +16,22 @@ export interface AppLookupParams {
 }
 
 async function ensureBundleIdExistsAsync(
+  ctx: AuthCtx,
```

Comment:
```suggestion
  authCtx: AuthCtx,
```
There is `context` in the same scope (just a few lines below). I think those variables shouldn't have so similar names.
The same comment applies to other functions that have both `AuthCtx` and `RequestContext` objects in their scope. 

(And yes, I've noticed that your convention is to name `AuthCtx` objects `ctx` and `RequestContext` objects `authCtx` but it takes some time to realize that. I think we should be as explicit as possible and this doesn't seem to be a big change.)